tools/pq/README.md

```diff
@@ -0,0 +1,32 @@
+# pq
```

Comment:
probably it would be nice to link here from poison queue godoc?

Reply:
Which one, though? The SQL one?

Reply:
Will add more docs in a follow-up PR.